### DISCUSSION TITLE
Item not getting highlighted with purple border in stash when getting whispers to buy it.

### DISCUSSION THREAD

**Discussion by freddiehaddad** [SnosMe/awakened-poe-trade#1097] [Q&A]
When I get a whisper to buy an item I have listed for sale, the item used to get highlighted with a purple border in my stash.  That's not happening anymore.  Is there an app setting to disable it or possibly an in-game graphics setting that's causing this? It used to work and now it doesn't.

**Comment by SnosMe** [comment]
This is a game built-in feature, not this app. irc this happens when people whisper you using bulk search, and it doesn't include item position in stash.

**Comment by freddiehaddad** [comment]
Hm.. I swear I noticed it after installing awakened poe trade.. guess that was just a coincidence that I noticed it.  Strange that its not happening anymore just for me.  I've asked in-game and people say the border shows up for them.  Thank you for letting me know it's not a feature of this app.

**Comment by w355re** [comment]
> When I get a whisper to buy an item I have listed for sale, the item used to get highlighted with a purple border in my stash. That's not happening anymore. Is there an app setting to disable it or possibly an in-game graphics setting that's causing this? It used to work and now it doesn't.

i have same trouble u fix it?

**Reply by freddiehaddad** [reply]
I didn't fix it, but I did reinstall POE on a new computer and it started working again. I can't explain what changed in the new install, but I can confirm it's not a feature of awakened poe trade.  You could try completely resetting all POE settings by deleting the config files.
